src/main/scala/KafkaSparkCassandra.scala

```diff
@@ -98,5 +102,6 @@ object KafkaSparkCassandra {
     rdd1.take(100).foreach(println)
     sc.stop()
 
+    System.exit(0)
```

Comment:
The exit is there because otherwise the program finishes but doesn't exit. Haven't been able to find out what's causing it as all the spark contexts etc are stopped from what I can see when debugging.